src/main/res/xml/svorak.xml

```diff
@@ -1,11 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 
-<Keyboard xmlns:android="http://schemas.android.com/apk/res/android"
-    android:keyWidth="9.05%p"
```

Comment:
Actually, I think you need to keep `android:keyWidth="9.05%p"` here. The `keyHeight` and the `gap`s can go.
Without the `keyWidth` the last key will be off-screen.